dash/quorum/selectpeers/dip6.go

```diff
@@ -0,0 +1,67 @@
+// Package selectpeers is package contains algorithm that selects peers based on the deterministic connection
+// selection algorithm described in DIP-6
+package selectpeers
+
+import (
+	"fmt"
+	"math"
+
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+	"github.com/tendermint/tendermint/types"
+)
+
+// minValidators is a minimum number of validators needed in order to execute the selection
+// algorithm. For less than this number, we connect to all validators.
+const minValidators = 5
+
+// DIP6 selects validators from the `validatorSetMembers`, based on algorithm
+// described in DIP-6 https://github.com/dashpay/dips/blob/master/dip-0006.md
+func DIP6(
```

Comment:
I think we should call this selectValidators, and add a parameter to choose how to select validators as an enum, right now we only have DIP6.

Reply:
Or something else is fine too. At least it feels weird calling this DIP6 function, thoughts?

Reply:
What about a more object-oriented approach, like https://github.com/dashevo/tenderdash/commit/e06c9f24c5d2801f4b9e3233587b400198f6801c , with interface ValidatorSelector that should be implemented by all algorithms:

https://github.com/dashevo/tenderdash/blob/e06c9f24c5d2801f4b9e3233587b400198f6801c/dash/quorum/selectpeers/validatorselector.go#L6-L9

Reply:
Sounds good